cmd/vshn-postgres-func/main.go

```diff
@@ -12,7 +12,7 @@ import (
 
 var postgresFunctions = []runtime.Transform[vshnv1.VSHNPostgreSQL, *vshnv1.VSHNPostgreSQL]{
 	{
-		Name:          "dummy",
+		Name:          "url",
```

Comment:
Nit: maybe a more descriptive name for the function could be helpful. E.g. `url-connectiondetail` or something like that.